dependency_name_overrides.yml

```diff
@@ -7,4 +7,4 @@
 # there defines just "slam3d".
 # To have the manually set package name occur in auto-generated package.xml files, you can define the 
 # name conversion from path to the manula package name here:
-slam/slam3d: slam3d
\ No newline at end of file
+#slam/slam3d: slam3d
```

Comment:
This should be kept, this is the translation override when generating package.xml files from manifest.xml files when importing rock packages to colcon using colcon_import_package